src/LinAlg/hiopMatrixDenseRowMajor.cpp

```diff
@@ -237,15 +237,15 @@ void hiopMatrixDenseRowMajor::copyFromMatrixBlock(const hiopMatrixDense& srcmat,
 void hiopMatrixDenseRowMajor::shiftRows(size_type shift)
 {
   if(shift==0) return;
-  if(fabs(shift)==m_local_) return; //nothing to shift
+  if(abs(shift)==m_local_) return; //nothing to shift
```

Comment:
I believe everything that is outside RAJA loops can use `std::abs`. For inside RAJA loops, it is better to use `fabs` or similar as CUDA (and I believe HIP, as well) stick to C interface for math functions.

Reply:
For inside RAJA loops, see #244.

Reply:
Since it seems like #244 has not seen much movement, I'll use naked `(f)abs` for RAJA and `std::` qualified `(f)abs` elsewhere. I'd like to resolve this in this PR.